spec.html

```diff
@@ -27868,10 +27867,9 @@ <h1>The String Constructor</h1>
         <h1>String ( _value_ )</h1>
         <p>When `String` is called with argument _value_, the following steps are taken:</p>
         <emu-alg>
-          1. If no arguments were passed to this function invocation, let _s_ be `""`.
-          1. Else,
-            1. If NewTarget is *undefined* and Type(_value_) is Symbol, return SymbolDescriptiveString(_value_).
-            1. Let _s_ be ? ToString(_value_).
+          1. If _value_ is not present, set _value_ to `""`.
+          1. If NewTarget is *undefined* and Type(_value_) is Symbol, return SymbolDescriptiveString(_value_).
+          1. Let _s_ be ? ToString(_value_).
```

Comment:
This call used to not happen when no arguments were passed; now it happens unconditionally. I'm not sure this is an improvement. Why de-nest these steps?